fuel-core/src/database.rs

```diff
@@ -148,6 +150,13 @@ impl Database {
         column: ColumnId,
         value: V,
     ) -> Result<Option<V>, Error> {
+        #[cfg(feature = "prometheus")]
+        {
+            DATABASE_METRICS.write_meter.inc();
+            DATABASE_METRICS
+                .bytes_written_meter
+                .inc_by(bincode::serialized_size(&value).unwrap_or(0));
```

Comment:
would it be more efficient to first perform the serialization and assign to a tmp var, then check the length of the serialized data for metrics afterwards?

Reply:
Yes that would be much better rather than doubling serialization times, just pushed a commit which addresses that